### ISSUE TITLE
The Near website is not editable

### ISSUE THREAD

**Issue by fulldecent** [near/NEPs#286]
Please put the Near website on GitHub and add a "edit this page" button to the bottom of each page.

There was something seriously wrong on your homepage. But I already forgot what it was in the time I was trying to tell you about it. In that same amount of time I could have just fixed it.

**Comment by fulldecent** [comment]
yes, please

**Comment by jakestutzman** [comment]
@fulldecent the near.org website is not officially open source for the community to edit at random. If you have feedback on the site, I would love to hear more. Please send any feedback here: https://forms.gle/7p2XfoznRhrJn9FU7 Thanks for caring!

**Comment by fulldecent** [comment]
Yes, editorial control of NEAR should be the responsibility of appointed NEAR people.

However, NEAR is open to feedback. And an EDIT button on GitHub may be much more effective rather than, on in addition to, that kind of link.

---

Alternative point:

If the website is designed in a way that near.org source contains something private then the website is designed wrong. Therefore I am assuming everything inside the near.org website is public. And it is a community project, so it should be open source by default.

**Comment by nearbuild** [comment]
agreed
